dali/base/dasds.cpp

```diff
@@ -6023,6 +6167,7 @@ CCovenSDSManager::CCovenSDSManager(ICoven &_coven, IPropertyTree &_config, const
     clientProps->setPropBool("@serverIterAvailable", true);
     clientProps->setPropBool("@useAppendOpt", true);
     clientProps->setPropBool("@serverGetIdsAvailable", true);
+    clientProps->setPropBool("@saveBinary", config.getPropBool("@saveBinary", false));
```

Comment:
For now, so it exercises this code a bit, I think we should turn on by default in debug builds.
Related, would be better to move line 6214 up, so reading from config once and applying default, then using 'saveBinary' here.